### ISSUE TITLE
2.79 + an-master ( not in addon list )

### ISSUE THREAD

**Issue by ionicartist** [JacquesLucke/animation_nodes#804]
I tried this on several machines now both at work and home... and every time I've tried to install from file animation nodes doesn't show on the addon list. I tried to move the files out of the subfolder into the main 'an's folder and that made it show up on the list but then it errors to there being multiple copies of 'an'... did i miss something?

**Comment by JacquesLucke** [comment]
https://github.com/JacquesLucke/animation_nodes/issues/797

(delete all AN folders before installing it again)

**Comment by ionicartist** [comment]
Jacque! first thank you so much for this powerful tool...

...So there was no prior version of AN in the addon folder... was a fresh install... and after each try i proceeded to delete the last.

**Comment by JacquesLucke** [comment]
I think you just downloaded the source code (using the download zip button) and not an actual build from the release page:
https://github.com/JacquesLucke/animation_nodes/releases/tag/v2.0
this is also what I suggested on the other site that I linked before.

I see that you are doing something wrong because you have `-master` in the file name. When you download a compiled release, this is not part of the name anymore.
